GraphQL-TSC.md

```diff
@@ -211,7 +211,7 @@ Each repository will be managed by the EasyCLA signature tool, ensuring that all
 
 GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). However when speaking on behalf of the GraphQL project (e.g. technical docs, project websites, READMEs, or anything merged into a git repository under the GraphQL organization) we do not allow posting opinions or messages of political advocacy using the voice of the project without prior approval via GraphQL TSC vote.
```

Comment:
I’m still stuck on second sentence. If the scope of the code of conduct includes outside of GitHub, when speaking as voice of project only, I think you need an “In addition,” and not a “however.” I would also add an example in the list from outside GitHub. And, minor, I would also change “we do not allow” to “we allow with permission,” just style, keeping it positive :)

Reply:
Good suggestions. I just dropped the "However", it's not bringing any clarity. I rephrased the latter to avoid a negative phrasing.